### PR TITLE
login and signup page with validation

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR features the following:
1. Login Page
2. Signup Page
3. There's is validation for email id and password
4. It features Show/Hide Password

login
sign-1
sign-2

### CODE REVIEW

File: mockbee-test/ecom-store/src/components/Auth/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  };
+
+  return (
+    <div className="d-grid">
+      <div className="login-auth direction-column d-flex justify-center">
+        <h2 className="auth-title">Signup</h2>
+        <div className="form-container">
+          <label className="form-label">Firtsname</label>
+          <input
+            type="email"
```
Comment (SanketDhabarde) [inline mockbee-test/ecom-store/src/components/Auth/SignUp.jsx]: type is text

File: mockbee-test/ecom-store/src/components/Auth/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            className="form-input lh-ls"
+            placeholder="Prakash"
+          />
+        </div>
+        <div className="form-container">
+          <label className="form-label">Last Name</label>
+          <input
+            type="email"
```
Comment (SanketDhabarde) [inline mockbee-test/ecom-store/src/components/Auth/SignUp.jsx]: here as well

**Review by maitrakhatri — APPROVED** [review]
logic code is clean and very easy to understand
good folder structure

**Review by SanketDhabarde — APPROVED** [review]
- like the folder structure. 
- logic looks simple and easy to understand
- learned about password validation from your code

File: mockbee-test/ecom-store/src/components/Auth/SignUp.jsx (hunk 1 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/components/Auth/SignUp.jsx]: thank you for pointing it out, I have updated it.

File: mockbee-test/ecom-store/src/components/Auth/SignUp.jsx (hunk 2 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/components/Auth/SignUp.jsx]: updated, thanks

**Pull request merged**